locales/ar-EG.json

```
@@ -207,49 +207,56 @@
     "update_available_short": "تحديث Elk",
     "webmanifest": {
       "canary": {
-        "description": "A nimble Mastodon web client (canary)",
-        "name": "Elk (canary)",
-        "short_name": "Elk (canary)"
+        "description": "نسخة ويب رشيقة ل Mastodon (النسخة الإنشائية)",
+        "name": "Elk (النسخة الإنشائية)",
+        "short_name": "Elk (النسخة الإنشائية)"
       },
       "dev": {
-        "description": "A nimble Mastodon web client (dev)",
-        "name": "Elk (dev)",
-        "short_name": "Elk (dev)"
+        "description": "نسخة ويب رشيقة ل Mastodon (النسخة التطويرية)",
+        "name": "Elk (النسخة التطويرية)",
+        "short_name": "Elk (النسخة التطويرية)"
       },
       "preview": {
-        "description": "A nimble Mastodon web client (preview)",
-        "name": "Elk (preview)",
-        "short_name": "Elk (preview)"
+        "description": "نسخة ويب رشيقة ل Mastodon (معاينة)",
+        "name": "Elk (معاينة)",
+        "short_name": "Elk (معاينة)"
       },
       "release": {
-        "description": "A nimble Mastodon web client",
+        "description": "نسخة ويب رشيقة ل Mastodon",
         "name": "Elk",
         "short_name": "Elk"
       }
     }
   },
   "search": {
     "search_desc": "ابحث عن الأشخاص والهاشتاج",
-    "search_empty": "لا يمكن العثور على أي شيء لشروط البحث هذه"
+    "search_empty": "لم يتم العثور على أي نتائج لشروط البحث الخاصة بك"
   },
   "settings": {
     "about": {
-      "label": "بشأن Elk"
+      "label": "حول Elk",
+      "meet_the_team": "تعرف على فريقنا",
+      "sponsor_action": "تمويل Elk",
+      "sponsor_action_desc": "لتمويل تطوير Elk والمساعدة في تحسينه",
+      "sponsors": "الرعاة",
+      "sponsors_body_1": "تم تمويل Elk من قبل الشركات والأفراد التاليين:" ,
```

Hello, @mininaim you can just remove the space just before `,` at the end of the line:
```suggestion
      "sponsors_body_1": "تم تمويل Elk من قبل الشركات والأفراد التاليين:",
```